react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/views/WrappedCustomerCenterView.kt

```diff
@@ -15,11 +20,34 @@ class WrappedCustomerCenterView(context: Context) : ComposeViewWrapper<CustomerC
         wrappedView?.setDismissHandler(dismissHandler)
     }
 
+    fun registerCustomerCenterListener(listener: CustomerCenterListener) {
+        val purchases = Purchases.sharedInstance
+        if (customerCenterListener == null) {
+            previousCustomerCenterListener = purchases.customerCenterListener
```

Comment:
what's this `previousCustomerCenterListener` for?

Reply:
I've removed this. It was over-engineering with no real reason 🙃